.travis.yml

```diff
@@ -10,19 +10,24 @@ os:
   - osx
 
 go:
-- "1.12"
+  - "1.12"
```

Comment:
The previous indentation should've been fine -- YAML has an explicit provision that allows lists to avoid the extra indent.
Please revert that part of the change, since it makes it easier to examine history.

Reply:
Done.